src/test/java/com/project/shop/ordersheet/domain/OrderSheetTest.java

```diff
@@ -0,0 +1,79 @@
+package com.project.shop.ordersheet.domain;
+import com.project.shop.item.domain.Category;
+import com.project.shop.item.domain.Item;
+import org.junit.jupiter.api.*;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class OrderSheetTest {
+    private LocalDateTime dateTime = LocalDateTime.now();
+
+    @DisplayName("주문서 생성 테스트")
+    @Test
+    public void create(){
+
+        Category 카테고리 = new Category("운동화", "나이키",dateTime);
+        Item 상품 = new Item(카테고리, "조던", 100000,"한정판매",dateTime);
+
+        OrderSheetItem 첫번째_주문상품 = new OrderSheetItem(상품, 1L, 10000,2);
+        OrderSheetItem 두번째_주문상품 = new OrderSheetItem(상품, 2L, 4000, 3);
+
+        List<OrderSheetItem> 상품리스트 = Arrays.asList(첫번째_주문상품,두번째_주문상품);
+        OrderSheet orderSheet = new OrderSheet(상품리스트);
+
+        assertThat(orderSheet).isInstanceOf(OrderSheet.class);
+    }
+
+    @DisplayName("생성 테스트 - 주문 상품 갯수(1개 이상)")
+    @Test
+    public void create_error_itemList(){
+
+        List<OrderSheetItem> 상품리스트 = Collections.emptyList();
+
+        Exception exception = assertThrows(IllegalArgumentException.class,
+                () -> new OrderSheet(상품리스트));
+
+        assertThat(exception.getMessage()).isEqualTo("주문 상품은 최소 1개 이상이여야한다.");
+
+    }
+
+    @DisplayName("주문서 등록 테스트")
+    @Test
+    public void createOrderSheet(){
+
+        Category 카테고리 = new Category("운동화", "나이키",dateTime);
+        Item 상품 = new Item(카테고리, "조던", 100000,"한정판매",dateTime);
+
+        OrderSheetItem 첫번째_주문상품 = new OrderSheetItem(상품, 1L, 10000,2);
+        OrderSheetItem 두번째_주문상품 = new OrderSheetItem(상품, 2L, 4000, 3);
+
+        List<OrderSheetItem> 상품리스트 = Arrays.asList(첫번째_주문상품,두번째_주문상품);
+        OrderSheet orderSheet = new OrderSheet(상품리스트);
+
+        orderSheet.createOrderSheet(상품리스트);
+
+        assertEquals(상품리스트.size(), orderSheet.getItemList().size());
+        assertEquals(상품리스트.get(1).getOrderQuantity(), 3);
```

Comment:
매직넘버 3이 보이네요.
```suggestion
        assertEquals(상품리스트.get(1).getOrderQuantity(), 두번째_주문상품_수량);
```

상품은 두개를 등록했는데 검증은 하나만 한 것 같아요.
첫번째_주문상품에 대한 수량이 틀렸다면 이 테스트의 신뢰도가 떨어질것같아요.